crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -2210,3 +2210,9 @@ impl WalletData for hyperswitch_domain_models::payment_method_data::WalletData {
         }
     }
 }
+
+/// Trait for converting from one foreign type to another
+pub trait ForeignFrom<F> {
+    /// Convert from a foreign type to the current type
+    fn foreign_from(from: F) -> Self;
+}
```

Comment:
Please remove this trait. It is not required.

Reply:
removed